README.md

```diff
@@ -6,3 +6,13 @@ Download the free versions from https://dev.maxmind.com/geoip/geoip2/geolite2/.
 
 * [Introduction to HAProxy ACLs](https://www.haproxy.com/blog/introduction-to-haproxy-acls/)
 * [Dovecot and HAProxy](https://wiki.dovecot.org/HAProxy)
+
+
+## Running
+
+```sh
+git clone https://github.com/andyjack/geoip2-haproxy-ranges
+cd geoip2-haproxy-ranges
+go get github.com/oschwald/maxminddb-golang
+go run main.go --db path/to/GeoLite2-Country.mmdb --destDir output
```

Comment:
Thanks for this pull request!

`go run . --db ...` (for go versions >= 1.11) will work, and then merging `flags.go` into `main.go` would no longer be necessary.

Reply:
ah that makes a ton of sense, I'll update it

Reply:
Hmm, Go complains:

```sh
$ go run . --db /dev/null --destDir output

go: cannot find main module, but found .git/config in /home/johannes/git/geoip2-haproxy-ranges
        to create a module there, run:
        go mod init
```

Reply:
Do you have the `GO111MODULE=on` environment variable set?  I don't think setting it to `on` is generally necessary these days.

```
GO111MODULE=on go run . -db /dev/null -destDir /tmp
go: cannot find main module, but found .git/config in /home/andy/local/work/geoip2-haproxy-ranges
        to create a module there, run:
        go mod init
```

```
# same result with GO111MODULE=off
GO111MODULE=auto go run . -db /dev/null -destDir /tmp
2021/03/04 15:09:01 invalid argument
exit status 1
```

Reply:
Interesting - I did not set `GO111MODULE`, but setting it to `auto` or `off` fixes it

here is the output of my `go env`

```sh
GO111MODULE=""
GOARCH="amd64"
GOBIN=""
GOCACHE="/home/johannes/.cache/go-build"
GOENV="/home/johannes/.config/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOINSECURE=""
GOMODCACHE="/home/johannes/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="linux"
GOPATH="/home/johannes/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/usr/lib/go"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/usr/lib/go/pkg/tool/linux_amd64"
GOVCS=""
GOVERSION="go1.16"
GCCGO="gccgo"
AR="ar"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/dev/null"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build2154090327=/tmp/go-build -gno-record-gcc-switches"
```